### DISCUSSION TITLE
Past Episode Air Dates in Episode List

### DISCUSSION THREAD

**Discussion by mtalexan** [trakt/showly#491] [Ideas / Feature Requests]
The Season lists of Episodes show upcoming air dates for episodes that haven't aired yet, but don't list air dates of past episodes. It's often useful to know when past episodes aired as well, without having to drill down into each individual episode.
For example, some shows have had long gaps between seasons (e.g Arrested Development), inconsistent season timelines (e.g. Rick and Morty), or Specials that can be from vastly different times throughout the series.

This feature would also help with cases where a number of episodes aired recently in a short period of time. As an alternative to viewing past episodes in the Calenders (separate discussion), seeing dates on past episodes would allow quick identification of those recently aired.

In implementation the past air dates should probably be a different color than upcoming air dates. This would preserve the clear indication of what has and hasn't aired.

The difference can be seen in the screenshot where the last episode in the list hasn't aired, but the next to last aired earlier in the day.
![Screenshot_20210531-213406.jpg](https://user-images.githubusercontent.com/7081950/120268389-8fc79900-c25a-11eb-8e8a-cdb0a095e113.jpg)

**Comment by michaldrabik** [comment]
I suppose it won't be a problem to just display this date on every item and not just upcoming ones.
